UnitsNet.Tests/CustomCode/HeatTransferCoefficientTests.cs

```diff
@@ -30,6 +30,7 @@ public class HeatTransferCoefficientTests : HeatTransferCoefficientTestsBase
         protected override bool SupportsSIUnitSystem => false;
         protected override double WattsPerSquareMeterCelsiusInOneWattPerSquareMeterKelvin => 1;
         protected override double WattsPerSquareMeterKelvinInOneWattPerSquareMeterKelvin => 1;
-        protected override double BtusPerSquareFootDegreeFahrenheitInOneWattPerSquareMeterKelvin => 1.7611018368230584e-1;
+        protected override double BtusPerHourSquareFootDegreeFahrenheitInOneWattPerSquareMeterKelvin => 1.7611018368230584e-1;
+        protected override double KilocaloriesPerHourSquareMeterDegreeCelsiusInOneWattPerSquareMeterKelvin => 8.6042065009560229e-1;
     }
```

Comment:
I find 0.859845227859 from
https://www.unitsconverters.com/en/W/M2mk-To-Kcal(It)/Hmm2mdegc/Utu-4398-4407

Reply:
0.85986
http://www.endmemo.com/sconvert/w_m2kcal_hm2.php

Reply:
Okay it seems there are TH and IT definitions of kilocalories as well.

Your definition is TH, but I believe all our existing BTU and calorie units use the IT definition.

https://www.translatorscafe.com/unit-converter/en-US/thermal-conductivity/1-7/watt/meter/K-kilocalorie%20(th)/hour/meter/%C2%B0C/?value=1

Reply:
It seems the convention is to use specify IT and TH for these units to be explicit. Maybe we should as well, it is very confusing to get slightly different values if mistakenly choosing IT vs TH units.

Reply:
To summarize; I propose 
- Change definition to IT calories (`KilocaloriesPerHourSquareMeterDegreeCelsiusInOneWattPerSquareMeterKelvin => 0.859845227859`)
- Add TODO to go over all BTU/calorie units and add IT/th suffix in #1200

Reply:
Yes, IT for calories in a heat transfer coefficient is appropriate, let's go with it. However, in that case I think we can use instead the official "cal(IT)" definition which is defined as 4.1868 J exactly:

"FromUnitToBaseFunc": "({x} * 4.1868) / 3600",
"FromBaseToUnitFunc": "({x} / 4.1868) * 3600",

[https://en.wikipedia.org/wiki/Calorie](url)

Regarding IT and TH consistency, Energy.json uses calories TH definition, while in HeatFlux.json IT is followed instead.

Reply:
Added TODO to #1200